raiden-dapp/src/views/TransferRoute.vue

```diff
@@ -64,6 +64,7 @@ export default class TransferRoute extends Vue {
       link: this.$t('notifications.backup-state.link') as string,
       dappRoute: RouteNames.ACCOUNT_BACKUP,
       description: this.$t('notifications.backup-state.description') as string,
+      duration: 60000,
```

Comment:
Could you use a 'dismiss' button on the notification as an alternative?

Reply:
There is already one, it can be dismissed by pressing the `X`.
<img width="408" alt="Screenshot 2020-11-09 at 13 31 14" src="https://user-images.githubusercontent.com/43838780/98541562-ebffa700-228f-11eb-8297-2a4ee8609063.png">

Reply:
My idea was that the thing would stay open until you dismiss it.

Reply:
60 seconds should be ok. Let's give it a try.

Reply:
@palango has a point though. If we keep it indefinitely the user would have to make an active decision to ignore it.

Reply:
Going with 60 sec as decided on the daily.